src/common.css

```diff
@@ -0,0 +1,4 @@
+.column {
+    display: flex;
+    flex-direction: column;
+}
```

Comment:
You'd better inline it in the places you use. Display flex and flex-direction is nothing special, in fact it's possible you will have it there already when you want to use it.

Reply:
It depends how many times you want to use specific group of styles together; he may be saving himself from duplication thanks to this, it all depends though how you use it. [I recommend this talk about systemic approach to css](https://www.youtube.com/watch?v=y2j6r5V67bY)

Reply:
:+1:  I deleted this for now. Maybe I will use this where I will have more uses.